SpatialGDK/Source/SpatialGDKTests/Examples/SpatialGDKTestGuidelines.h

```diff
@@ -0,0 +1,82 @@
+/*
+1. How to run tests:
+	- Tests can be run via Session Frontend in UE4 Editor
+	(https://docs.unrealengine.com/en-US/Programming/Automation/index.html has more information)
+
+	- Tests can be run via command line:
+	{PathToUnreal}\Engine\Binaries\Win64\UE4Editor-Cmd.exe ^
+	 "{PathToUProjectFile}.uproject" ^
+	 -unattended -nopause -NullRHI -log -log=RunTests.log ^
+	 -ExecCmds="Automation RunTests {TestFilter}; Quit"
+
+	where {TestFilter} is a match on the test names
+	(e.g SpatialGDK matches all GDK tests, FRPCContainer matches all tests in RPCContainerTest)
+
+2. Folder structure
+	- Testing code that is not a part of exposed API
+		- These tests should go into the Private\Tests directory within the relevant module.
+		When an Automation Test matches one-to-one with a particular class, the test file should be named [ClassFilename]Test.cpp,
+		e.g. a test that applies only to FText would be written in TextTest.cpp.
+
+	- Testing code that is a part of exposed API or Integration tests
+		- These tests are located in a separate SpatialGDKTests module.
+		So for every component that needs to be tested - it has to be exposed via corresponding macro.
+		E.g. `class FRPCContainer` -> `class SPATIALGDK_API FRPCContainer`, to make FRPCContainer testable.
+		WThe macro is different in each module SPATIALGDK_API, SPATIALGDKSERVICES_API etc.
+
+		- The folder structure inside SpatialGDKTests should resemble the folder structure for components being tested.
+		E.g. If file tested is
+		`UnrealGDK\SpatialGDK\Source\SpatialGDK\Public\Utils\RPCContainer.cpp`,
+		then the corresponding test should be located in
+		`UnrealGDK\SpatialGDK\Source\SpatialGDKTests\SpatialGDK\Utils\RPCContainer\RPCContainerTest.cpp`.
+		There should be a folder with the same name as component tested. It should include the test and all the supporting files.
+
+		- In case of integration tests, that involve multiple components - they should be located in
+		`UnrealGDK\SpatialGDK\Source\SpatialGDKTests\IntegrationTests\<Custom Test Folder>` folder.
+
+3. Test definitions (check TestDefinitions.h for more info)
+    - We have defined 3 types of Macro to be used when writing tests:
+	(https://docs.unrealengine.com/en-US/Programming/Automation/TechnicalGuide/index.html has more information)
+	TEST - a simple test, that should be used if it's one of a kind (i.e. it's body can't be reused, otherwise use COMPLEX_TEST),
```

Comment:
```suggestion
	GDK_TEST - a simple test, that should be used if it's one of a kind (i.e. it's body can't be reused, otherwise use GDK_COMPLEX_TEST),
```